### PR TITLE
Add missing LICENSE files

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tickets       | -
| License       | MIT

### CODE REVIEW

**Review by nicolas-grekas — APPROVED** [review]

**Review by OskarStark — APPROVED** [review]
Thanks

**Review by fabpot — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

File: src/LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2020-2021 Fabien Potencier
```
Comment (stof) [inline src/LICENSE:1]: This one should be at the root, not in `src/`

Comment (chalasr) [reply src/LICENSE:1]: thanks, fixed in 57e02174f961aa537b58428ad7edecf12d879a5e

**Pull request merged**